src/Driver.ts

```diff
@@ -58,3 +57,21 @@ export const DRIVERS = {
   MEMORY: new Driver(new MemoryStorage()),
   COOKIE: new Driver(new CookieStorage())
 }
+
+export function determineDriver(defaultDriverType?: any): Driver {
+  let hasFallback = Array.isArray(defaultDriverType)
```

Comment:
This removes ability to set single default driver type. Ideally we preserve both